Initium-Odp/war/odp/javascript/script.js

```diff
@@ -4916,21 +4859,42 @@ $(document).keyup(function(event){
 	{
 		viewProfile();
 	}
-	else if (event.which==76) // Changed to L since B is now for Nearby Characters list
-	{
-		window.location.href='main.jsp';
+	//user can just refresh the page. besides, this breaks on experimental.
+	//else if (event.which==76) // Changed to L since B is now for Nearby Characters list
+	//{
+	//	window.location.href='main.jsp';
+	//}
+	else if (event.which==85){ // U - go up in character list
+		switchCharacter(event, null, 4);
```

Comment:
switchCharacter just seems to be repeated a lot with no differences between hotkeys